include/glow/Backends/BackendOptions.h

```diff
@@ -16,7 +16,19 @@
 #ifndef GLOW_BACKENDS_BACKENDOPTIONS_H
 #define GLOW_BACKENDS_BACKENDOPTIONS_H
 
+#include "llvm/ADT/SmallVector.h"
+
 namespace glow {
+class Storage;
+
+/// Hints provided to the Backend, the backend is not required to honor them.
+struct BackendHints {
+  /// Number of execution units to reserve, 0 for unspecified.
+  unsigned executionUnits{0};
+
+  /// Storage nodes to be pinned to SRAM listed in order of priority.
+  llvm::SmallVector<Storage *, 0> SRAMPrioritization;
```

Comment:
Can we design a way for this to be manually input to allow for experimentation before finalizing (just like with Partitioner?)

Reply:
Similar to user-defined partitioning? 
Would names make this easier to specify then?

Reply:
> Can we design a way for this to be manually input to allow for experimentation before finalizing (just like with Partitioner?)

That should be something in Partitioner.  After the Partitioner refactoring, I think it should be easy to add this feature in experimental flow.